lib/sidecar/process_supervisor.ex

```diff
@@ -10,9 +10,9 @@ defmodule Sidecar.ProcessSupervisor do
     children =
       [
         supervisor_process_logger(__MODULE__),
-        statestores(),
         {Sidecar.MetricsSupervisor, opts},
         Spawn.Supervisor.child_spec(opts),
+        statestores(),
```

Comment:
Statestores needs to be after libcluster to make it work

What are some possible implications you can think of @sleipnir ?

Reply:
Starting before the Actor.Entity supervisor is ok